### DISCUSSION TITLE
The unread dot overlaps with the timestamp in the "Request hasn't been accepted" notification

### DISCUSSION THREAD

**Discussion by smohamedjavid** [status-im/status-legacy#16681] [General]
## Bug Report

When a user requests to join the community and it goes unattended, A "Request hasn't been accepted" notification is received in the Activity Center of the user. The unread dot of the notification overlaps with the timestamp.

![Issue copy](https://user-images.githubusercontent.com/19339952/232018421-73327e0b-b8f9-4289-8bf7-7692aedddc00.png)

In [Figma](https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=4537%3A515620&t=KbCqCLW3kNt9LXy0-1), The timestamp is **missing** in this notification type, and this needs to be discussed with the design team.

#### Precondition: User `A`  is an admin of a closed community. 

- User `A` shares community with user `B`
- User `B` sends a "Request to join the community" 
- User `B` AC gets updated with a notification that he/she requested to join the community 
- User `A` receives a "Join Request" notification in AC and leaves the request unattended for 7 days 
- User `A` checks the "Join Request" notification in AC and it's removed (after 7 days)
- User `B` opens the AC and sees that the "Request hasn't been accepted"

### Possible Solution
The notification title can be a short one just as other notifications

**Comment by ibrkhalil** [comment]
[For designers]
I see there's horizontal spacing, Why can't we utilize it?

**Reply by cammellos** [reply]
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=1578-34575&mode=design&t=MxlL05YCAl41bl2I-0 this has been added now, Notifications -> along two lines
